src/test.ts

```diff
@@ -1,43 +1,74 @@
-// This file is a playground to test the function signatures
+import { expect } from "chai";
+import * as iobHelper from "./main";
 
-import { buildObject } from "./main";
-
-const iobObj1 = buildObject(undefined as any, {
-	id: "statistics.total_distance_meter",
-	name: "total_distance_meter",
-	value: "test",
-	objectType: "state",
-	role: "value",
-	description: "Total distance in meters",
-});
-iobObj1.object.common.unit = "m";
-
-const iobObj2 = buildObject(undefined as any, {
-	id: "statistics.total_distance_meter",
-	name: "total_distance_meter",
-	value: "test",
-	objectType: "template",
-	template: "alive",
-	description: "Total distance in meters",
-});
-iobObj2.object.common.unit = "m";
-
-const iobObj3 = buildObject(undefined as any, {
-	id: "statistics.total_distance_meter",
-	name: "total_distance_meter",
-	value: "test",
-	objectType: "template",
-	template: "device",
-	description: "Total distance in meters",
+//#region Test buildObject
+describe("Test buildObject", () =>{
+	it("Build channel object by template", () => {
+		const iobObjChannel1 = iobHelper.buildObject({ namespace: "test.0"} as globalThis.ioBroker.Adapter , {id: "channel1", name: "channel1", objectType: "template", template: "channel"});
+		//console.log(JSON.stringify(iobObjChannel1));
+		expect(iobObjChannel1).to.have.keys(["id", "object"]);
+		expect(iobObjChannel1.id).to.equal("test.0.channel1");
+		expect(iobObjChannel1.object).to.have.keys(["type", "common", "native"]);
+		expect(iobObjChannel1.object.common.name).to.equal("channel1");

```

Comment:
I think `.to.deep.equal({ ... })` would/could be clearer if we want to assert how the objects should look like.